src/Utils/language.utils.js

```diff
@@ -0,0 +1,31 @@
+import I18n from 'react-native-i18n'; // You can import i18n-js as well if you don't want the app to set default locale from the device locale.
```

Comment:
react-native-i18n is going to be deprecated. https://github.com/AlexanderZaytsev/react-native-i18n#%EF%B8%8F-this-package-is-deprecated